sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_aad.py

```diff
@@ -0,0 +1,48 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, WEBPUBSUB_ENDPOINT
+try:
+    endpoint = os.environ.get("WEBPUBSUB_ENDPOINT")
+except KeyError:
+    LOG.error("Missing environment variable 'WEBPUBSUB_ENDPOINT' - please set if before running the example")
+    exit()
+
+# Build a client from the connection string. And for this example, we have enabled debug
+# tracing. For production code, this should be turned off.
+client = WebPubSubServiceClient(credential=DefaultAzureCredential(), endpoint=endpoint)
```

Comment:
You're not building a client from the connection string here, even though the comments are saying that you are. Since you already have a separate sample for this, I think you should remove this and the following code from this sample

Reply:
my carelessness and I will improve it.